test/integration/webhook/clusterqueue_test.go

```diff
@@ -182,12 +184,35 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 		},
 			ginkgo.Entry("Should have non-negative borrowing limit",
 				testing.MakeClusterQueue("cluster-queue").
-					ResourceGroup(*testing.MakeFlavorQuotas("x86").Resource("2", "-1").Obj()).
+					ResourceGroup(*testing.MakeFlavorQuotas("x86").Resource("cpu", "2", "-1").Obj()).
```

Comment:
You can remove these tests. The unit tests should be enough. We had more of these tests in the past, before we added the unit versions.